src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -187,6 +236,7 @@ export default function DownloadListTable(props: DownloadListTableProps) {
                 </th>
                 <th>
                   SynID
+                  <span>{showInteractiveCopyIdsIcon()}</span>
                   <span>{showInteractiveSortIcon('synId')}</span>
```

Comment:
Just a style note, I would prefer these to be JSX elements rather than invoked functions